frodo.js

```diff
@@ -1,23 +1,23 @@
-const isPalindrome = word => word === word.split("").reverse().join("");
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?